var/spack/repos/builtin/packages/amber/package.py

```diff
@@ -104,6 +104,8 @@ class Amber(Package, CudaPackage):
         patch(patch_url_str.format(ver, num),
               sha256=checksum, level=0, when='@{0}'.format(ver))
 
+    # Patch to fix sebomd_module.F90
+    patch('sebomd_fix.patch', when='@20')
```

Comment:
Can you expand what "fix" means in this context, and add where does the patch come from?

Reply:
sure, the commit itself said where the fix came from but i can add it as a comment too

Reply:
> Where did you get the patch? If it is from the official patches, better to apply those. Further, this issue seems to apply only to a few compilers (check [Thread](http://archive.ambermd.org/202105/0086.html)), I would feel more comfortable applying an official patch instead of this explicit one.

That's outdated, but from here: http://archive.ambermd.org/202105/0098.html
There is no "official" patch; and this just said what fixed it so i created the patch myself. The error was encountered when using gcc@10 and gcc@11; honestly if you are supposed to use gcc@9 it's never said in all of the ambertools documentation